devops/infrastructure/bash-config.sample.cfg

```diff
@@ -33,3 +41,11 @@ BRANCH_NAME=sumer
 
 # If true will build LOCAL_CODE_PATH otherwise will pull from GIT_REPO:BRANCH_NAME
 BUILD_LOCAL_CODE=false
+
+get_aws_export () {
```

Comment:
oh its here :)
I think it would be better to leave this in a common file other than the cfg

Reply:
Stack name and export name are parameters now